code/modules/mob/living/carbon/emote.dm

```diff
@@ -125,11 +125,44 @@
 
 		if ("scream","screams")
 			if (!muzzled)
-				..(act)
+				var/sound = pick('sound/misc/scream_m1.ogg', 'sound/misc/scream_m2.ogg')
+
+				if(src.dna)
+					var/DNA = src.dna.species.id
+
+					switch(DNA)
+						if("IPC")
+							sound = "sound/voice/screamsilicon.ogg"
+						if("tarajan")
+							sound = "sound/misc/cat.ogg"
+						if("lizard")
+							sound = "sound/misc/lizard.ogg"
+						if("avian")
+							sound = "sound/misc/caw.ogg"
+						if("skeleton")
+							sound = "sound/misc/skeleton.ogg"
+						if ("moth")
+							sound = "sound/misc/moth.ogg"
+						else
+							if(gender == FEMALE)
+								sound = pick('sound/misc/scream_f1.ogg', 'sound/misc/scream_f2.ogg')
+				
+				if(isalien(src))
+					sound = pick('sound/voice/hiss6.ogg')
+				
+				if(alternate_screams.len > 0)
```

Comment:
Oh right, well it's defined, but rather if it's got a null value/wasn't initialised would it still return 0?